src/js/lib/idGenerator.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+let currId = 0;
```

Comment:
yea, if you notice its references as "localId" in most places. Its for react/redux mainly.